src/data/faq.json

```diff
@@ -1052,35 +1054,42 @@
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. Even when this functionality was still available in the luca app, luca app users were not automatically warned when CWA users tested positive.<hr>",
+                                    "Even if users of both apps scan the same QR code, no data flows from one system to the other. Only when the responsible health office was informed about the infection from a test laboratory, for example, it could have personally informed the affected luca users about the possible risk contact. However, the principle of the CWA is based on the fact that users are warned pseudonymously via a 'red tile' (increased risk) in the event of a risk encounter. No interaction with a health department is necessary in this process."
                                 ]
                             },
                             {
-                                "title": "Can all QR codes for check-in be read immediately by the Corona-Warn-App and the luca app?",
+                                "title": "Can all luca QR codes for check-in be read immediately by the Corona-Warn-App and the luca app?",
                                 "anchor": "check_in_luca_all",
                                 "active": false,
                                 "textblock": [
-                                    "The luca-QR codes generated as of 25 May 2021 can be read (scanned) by both the Corona-Warn-App and luca app and processed for the check-in process - unless the operators have explicitly contradicted this. In the case of an incompatible luca QR code, you can contact the organizer, who will issue a new compatible luca QR code if necessary. The support of private QR codes is still not possible."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. However, luca QR codes can still be scanned with the Corona-Warn-App.<hr>",
+                                    "The luca-QR codes generated as of 25 May 2021 can be read (scanned) by both the Corona-Warn-App and luca app (no longer supported since spring 2022) and processed for the check-in process - unless the operators have explicitly contradicted this. In the case of an incompatible luca QR code, you can contact the organizer, who will issue a new Corona-Warn-App QR code if necessary. The support of private luca QR codes is still not possible."

```

Comment:
"In the case of an incompatible luca QR code, you can contact the organizer, who will issue a new Corona-Warn-App QR code if necessary."

Is that still true?

Reply:
Removed that sentence in commit https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks